Tasks/ServiceFabricUpdateAppVersions/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -6,6 +6,7 @@
   "loc.input.label.applicationPackagePath": "Application Package",
   "loc.input.help.applicationPackagePath": "Path to the application package. [Variables](https://go.microsoft.com/fwlink/?LinkID=550988) and wildcards can be used in the path.",
   "loc.input.label.versionSuffix": "Version Value",
+  "loc.input.help.versionSuffix": "The value used to the specify the version in the manifest files. Default is .$(Build.BuildNumber).",
```

Comment:
Also, should we say that this value will be used as suffix. Actual version value will also depend on prefix specified in manifest files.